### ISSUE TITLE
Kodi Connect not working

### ISSUE THREAD

**Issue by jrbongers** [Sonarr/Sonarr#5453]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

Kodi library is not updated through Kodi Connect.
Kodi-standalone is running on same server,
Kodi is triggered but it looks like the wrong episode or file is being sent by sonarr.
Trigger is a new episode of NCIS, season 20.
Kodi Connect is working with latest Radarr.

K2023-02-25 14:49:52.800 T:254748 debug <CWebserver[8033]>: request received for /jsonrpc
2023-02-25 14:49:52.889 T:254750 info : Skipped 1 duplicate messages…
2023-02-25 14:49:52.889 T:254750 debug : Thread JobWorker start, auto delete: true
2023-02-25 14:49:52.902 T:254750 debug : This query part contains a like, we will double backslash in the next field: ’ AND idPath NOT IN (SELECT idPath FROM files WHERE strFileName LIKE ‘video_ts.ifo’) AND idPath NOT IN (SELECT idPath FROM files WHERE strFileName LIKE ‘index.bdmv’)
2023-02-25 14:49:52.939 T:216721 debug : ------ Window Init (DialogExtendedProgressBar.xml) ------
2023-02-25 14:49:52.946 T:254750 info : VideoInfoScanner: Starting scan …
2023-02-25 14:49:52.958 T:254750 warning : Process directory ‘smb://192.168.178.10/TV/NCIS/Season 18/’ does not exist - skipping scan and clean.
2023-02-25 14:49:52.960 T:254750 warning : Process directory ‘smb://192.168.178.10/TV/NCIS/Season 19/’ does not exist - skipping scan and clean.
2023-02-25 14:49:52.971 T:254750 info : CleanDatabase: Starting videodatabase cleanup 

### Expected Behavior

kodi l;ibrary update with latest episode.

### Steps To Reproduce

tested with import and downloaded episodes, same results.
test with latest episode 1923 in trace log
checked new download in radarr, library was updated.

### Environment

```markdown
- OS: ubuntu 22.04.2
- Sonarr:4.0.0.391
- Docker Install:
- Using Reverse Proxy:
- Browser:
```

### What branch are you running?

Develop

### Trace Logs?

[sonarr.trace.txt](https://github.com/Sonarr/Sonarr/files/10834723/sonarr.trace.txt)
[kodi.log](https://github.com/Sonarr/Sonarr/files/10834724/kodi.log)

**Comment by markus101** [comment]
Sonarr tells Kodi to refresh a series path, it doesn't tell it to refresh a season folder or episode.

This is the request for 1923 in the trace logs, I don't see NCIS, but shows the path Sonarr is sending:
```
2023-02-26 21:46:54.7|Trace|HttpClient|Req: [POST] http://localhost:8033/jsonrpc: VideoLibrary.Scan("smb://192.168.178.10/TV/1923/")
```
